src/app/visit/visit.cpp

```diff
@@ -42,13 +48,20 @@ void Visit::createVisit(const std::shared_ptr<Doctor>& doctor)
 {
     Visit visit(doctor);
     auto visit_ptr = std::make_shared<Visit>(visit);
```

Comment:
I was almost sure that this one was corrected already...:
Why we create local visit and then make copy to shared_ptr?
Can't we make it in one go via make_shared?
Perhaps there are some problems with private constructor, but it looks a bit strange in my opinion.